amundsen_application/static/js/components/common/ResourceListItem/SchemaInfo/index.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { OverlayTrigger, Popover } from 'react-bootstrap';
+
+import './styles.scss';
+
+export interface SchemaInfoProps{
+  schema?: string;
+  placement?: string;
+  desc?: string;
+}
+
+const SchemaInfo: React.SFC<SchemaInfoProps>  = ({ schema, placement, desc }) => {
+  if (schema === null || schema === '') {
+    return null;
+  }
+  const popoverHoverFocus = (
+   <Popover id="popover-trigger-hover-focus">
+     <strong>{ schema }:</strong> { desc }
+   </Popover>
+ );
+
+  return (
+    <OverlayTrigger
+     trigger={['hover', 'focus']}
+     placement={ placement }
+     overlay={popoverHoverFocus}
+     >
+      <span className="underline">{ schema }</span>
+    </OverlayTrigger>
+  );
+};
+
+SchemaInfo.defaultProps = {
+  schema: '',
+  placement: 'bottom',
+  desc: 'Main source of all events data, usually recommended for use, Data is usually 24-48 hours delayed.'
```

Comment:
I assume this is just being used for development purposes, but we won't want a default description here. We'd want to pass the `schema_description` from the API. 

So for the actual implementation there should be no default description. Instead we would have some boolean logic to render either the plain text or the text with the `OverlayTrigger`.

Reply:
yes it is just for development purpose, I will remove default values. I planning to keep only  placement = bottom as default.
yes you are right, I will put a boolean logic if we do not get any schema_description